### DISCUSSION TITLE
Major issues with 1.6.1

### DISCUSSION THREAD

**Discussion by SunDevil311** [aritra-tech/Notify#215] [Q&A] [answered]
I'm not sure what's going on, but I'll try to be brief. Shortly prior to the 1.6.1 update, I had toggled on the biometric lock option. Everything was working fine prior to updating.

After the update, I simply cannot get Notify to open. It will briefly show the fingerprint sensor (shows for <5 sec, probably measurable in ms). However, the program then abruptly crashes, triggering a Samsung app error notice (and continuing to run, it appears).

I've tried clearing all data (luckily not much in there yet) and uninstalling, then re-installing Notify 1.5.1. Still getting the same problem with both attempts.

I have plenty of other biometric-enabled apps that continue to run without issue.

Only thing I can think of is I recently configured NextDNS. I'm only using one nominally aggressive filter (HaGeZi - Multi NORMAL), but if it's authenticating (or attempting to) with a remote DB, the traffic may be getting blocked.

That's the only thing I can think of, nothing else has changed recently. Would've made a good replacement for Google Keep Notes, but this is causing me to reconsider.

Appreciate any advice/assistance you can give me!

Environment:
Samsung Galaxy S20 5G
Android 13 / OneUI 5.1
Notify installed with Droid-ify / F-Droid via IzzyOnDroid, as well as directly via APK for tshooting

~ SunDevil311 ~

**Comment by aritra-tech** [comment, marked as answer]
Thanks for pointing @SunDevil311.
I will take a look and try to fix it on the next update

**Reply by SunDevil311** [reply]
Much appreciated!

**Reply by aritra-tech** [reply]
@SunDevil311  please mark this as an answer if you have no further queries so that others can know that the question has been answered.
